### DISCUSSION TITLE
Displaying multiple labels and Catalogs properly?

### DISCUSSION THREAD

**Discussion by ghost** [TT-ReBORN/Georgia-ReBORN#61] [Q&A] [answered]
Hi, I was wondering how to get dividers for multiple labels and catalog numbers like how it is for genres? Example:
![image](https://user-images.githubusercontent.com/43250660/161156818-de71bbe7-ad77-4384-a1a7-067de7d95167.png)
Here is what they look like right now:
![image](https://user-images.githubusercontent.com/43250660/161155256-7685f3a0-2f2a-47a0-84cd-dc4870af359a.png)
And here are my metadata tags:
![image](https://user-images.githubusercontent.com/43250660/161155307-8cb7870f-820f-40bc-ba04-68bedb25c0d3.png)
Any help?

**Comment by silverwings15** [comment, marked as answer]
you need to right click on `<LABEL>` or `<CATALOGNUMBER>`, click on Split Values... and then OK

**Reply by ghost** [reply]
Thank you, this worked perfectly
